phpBB/phpbb/di/service_collection/service_collection.php

```diff
@@ -11,7 +11,7 @@
 *
 */
 
-namespace phpbb\di;
+namespace phpbb\di\service_collection;
```

Comment:
Can we avoid this? It breaks all extensions unnecessarily that use a service collection. Having:
* phpBB/phpbb/di/service_collection.php
* phpBB/phpbb/di/ordered_service_collection.php

is just as fine?